src/components/CCPForm/CCPFormPage.tsx

```diff
@@ -84,12 +89,29 @@ const CCPFormPage = ({
 
   const [addCCP] = useMutation(ADD_CCP, {
     update(cache, { data: { addCollectionPoint } }) {
```

Comment:
I'm a little unclear why there are three queries for adding a CCP. Could you please elaborate on this?

Reply:
I'm not sure if this is the clarification you need, but `cache.readQuery` reads from the cache, while `useQuery` actually attempts to fetch data from the backend if it's not found in the cache.